I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 153, in script
    play muisic "audio/megastructure1.mp3"
  File "renpy/common/000statements.rpy", line 123, in execute_play_music
    renpy.music.play(_audio_eval(p["file"]),
Exception: Audio channel 'muisic' is unknown.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 153, in script
    play muisic "audio/megastructure1.mp3"
  File "D:\PROGRAMAS\renpy\renpy-8.0.0-sdk\renpy\ast.py", line 2212, in execute
    self.call("execute")
  File "D:\PROGRAMAS\renpy\renpy-8.0.0-sdk\renpy\ast.py", line 2200, in call
    return renpy.statements.call(method, parsed, *args, **kwargs)
  File "D:\PROGRAMAS\renpy\renpy-8.0.0-sdk\renpy\statements.py", line 281, in call
    return method(parsed, *args, **kwargs)
  File "renpy/common/000statements.rpy", line 123, in execute_play_music
    renpy.music.play(_audio_eval(p["file"]),
  File "D:\PROGRAMAS\renpy\renpy-8.0.0-sdk\renpy\audio\music.py", line 103, in play
    c = get_channel(channel)
  File "D:\PROGRAMAS\renpy\renpy-8.0.0-sdk\renpy\audio\audio.py", line 915, in get_channel
    raise Exception("Audio channel %r is unknown." % name)
Exception: Audio channel 'muisic' is unknown.

Windows-10-10.0.18362 AMD64
Ren'Py 8.0.0.22062402
VlastVN 1.0
Mon Jul  4 01:25:10 2022
